code/modules/antagonists/hivemind/hivemind.dm

```diff
@@ -185,6 +186,26 @@
 	C.add_trait(TRAIT_NOHUNGER, HIVEMIND_ONE_MIND_TRAIT)
 	C.add_trait(TRAIT_NODISMEMBER, HIVEMIND_ONE_MIND_TRAIT)
 
+/datum/antagonist/hivemind/proc/go_back_to_sleep()
+	if(!active_one_mind)
+		return
+	for(var/datum/mind/M in hivemembers)
+		M.remove_antag_datum(/datum/antagonist/hivevessel)
+		active_one_mind.remove_member(M)
+	if(!owner?.current)
```

Comment:
is this order of operations friendly?

if not
```suggestion
	if(!(owner?.current))
```

Reply:
i actually don't know, good point